### ISSUE TITLE
loss suddenly increasing after some training steps

### ISSUE THREAD

**Issue by pashanitw** [RVC-Boss/GPT-SoVITS#1264]
Hello,
Thanks for the great codebase. i am working on training the model for hindi lang. i have 4k hours of training data. so i am training it from scratch. i changed the hubert model and using hindi bert models instead. then i am training the model from scratch s2_train. but loss suddenly increasing after some steps. i attached the screenshot of training losses below
![image](https://github.com/RVC-Boss/GPT-SoVITS/assets/1389446/49801826-18e7-4867-8664-f117eecd12fd)

**Comment by RVC-Boss** [comment]
when meeting loss suddenly increasing issue, maybe fp32 training can fix it

**Comment by pashanitw** [comment]
yes fp32 worked thanks !

**Comment by utkarshshukla2912** [comment]
@pashanitw were you able to train the model and how is the output experience for you

**Comment by Chi8wah** [comment]
@pashanitw hi may I ask how is the final result? Does it meet your expectations?
